portfolio/src/main/java/com/google/sps/data/ShortestTaskFirst.java

```diff
@@ -21,13 +21,9 @@
 import java.util.Comparator;
 import java.util.List;
 
-// TODO(tomasalvarez): Refactor this class into an abstract class where each of
-// the scheduling algorithms can be their own class that extends this one.
-// TODO: Add test coverage for scheduling algorithms
-
-public class FindSchedule {
+public class ShortestTaskFirst extends BaseTaskScheduler {
```

Comment:
Consider using suffix to indicate that this is a task scheduler.

ex:
ShortestTaskFirstScheduler

Reply:
Done.